src/pages/recipes/AddIngredient.jsx

```diff
@@ -18,40 +22,67 @@ function AddIngredient({ onSubmit }) {
 
   const navigate = useNavigate();
 
-  const handleBackButtonClick = () => {
+  const handleButtonClick = () => {
     navigate("/recipe/new");
   };
 
+  const isEditing = !!ingredientToEdit;
+
   return (
     <form onSubmit={handleSubmit(onSubmit)}>
-      <Text size="h3">Dodaj składnik</Text>
+      <Text size="h3">{isEditing ? "Edytuj" : "Zapisz"} składnik</Text>
+      {ingredientToEdit?.id && (
+        <Input
+          hidden
+          label="Id"
+          {...register("id")}
+          defaultValue={ingredientToEdit?.id}
```

Comment:
it will be always `undefined`, because if `ingredientToEdit?.id` is defined it won't be rendered